src/rust/engine/process_execution/bazel_protos/src/verification_tests.rs

```diff
@@ -16,109 +15,102 @@ const OTHER_DIRECTORY_SIZE: i64 = 0;
-    file.set_name("simba".to_owned());
-    file.set_digest({
-      let mut digest = Digest::new();
-      digest.set_size_bytes(FILE_SIZE);
-      digest.set_hash(HASH.to_owned());
-      digest
-    });
-    file
-  });
-  directory.mut_directories().push({
-    let mut dir = DirectoryNode::new();
-    dir.set_name("cats".to_owned());
-    dir.set_digest({
-      let mut digest = Digest::new();
-      digest.set_size_bytes(DIRECTORY_SIZE);
-      digest.set_hash(DIRECTORY_HASH.to_owned());
-      digest
-    });
-    dir
-  });
-  directory.mut_directories().push({
-    let mut dir = DirectoryNode::new();
-    dir.set_name("dogs".to_owned());
-    dir.set_digest({
-      let mut digest = Digest::new();
-      digest.set_size_bytes(OTHER_DIRECTORY_SIZE);
-      digest.set_hash(OTHER_DIRECTORY_HASH.to_owned());
-      digest
-    });
-    dir
-  });
-  assert_eq!(Ok(()), verify_directory_canonical(EMPTY_DIGEST, &directory));
-}
+  let directory = Directory {
+    files: vec![
+      FileNode {
+        name: "roland".to_owned(),
+        digest: Some(Digest {
+          hash: HASH.to_owned(),
+          size_bytes: FILE_SIZE,
+        }),
+        ..FileNode::default()
+      },
+      FileNode {
+        name: "simba".to_owned(),
+        digest: Some(Digest {
+          hash: HASH.to_owned(),
+          size_bytes: FILE_SIZE,
+        }),
+        ..FileNode::default()
+      },
+    ],
+    directories: vec![
+      DirectoryNode {
+        name: "cats".to_owned(),
+        digest: Some(Digest {
+          hash: DIRECTORY_HASH.to_owned(),
+          size_bytes: DIRECTORY_SIZE,
+        }),
+      },
+      DirectoryNode {
+        name: "dogs".to_owned(),
+        digest: Some(Digest {
+          hash: OTHER_DIRECTORY_HASH.to_owned(),
+          size_bytes: OTHER_DIRECTORY_SIZE,
+        }),
+      },
+    ],
+    ..Directory::default()
+  };
 
-#[test]
-fn unknown_field() {
-  let mut directory = Directory::new();
-  directory.mut_unknown_fields().add_fixed32(42, 42);
-  let error = verify_directory_canonical(EMPTY_DIGEST, &directory).expect_err("Want error");
-  assert!(
-    error.contains("unknown"),
-    format!("Bad error message: {}", error)
-  );
+  assert_eq!(Ok(()), verify_directory_canonical(EMPTY_DIGEST, &directory));
 }
 
-#[test]
-fn unknown_field_in_file_node() {
-  let mut directory = Directory::new();
-
-  directory.mut_files().push({
-    let mut file = FileNode::new();
-    file.set_name("roland".to_owned());
-    file.set_digest({
-      let mut digest = Digest::new();
-      digest.set_size_bytes(FILE_SIZE);
-      digest.set_hash(HASH.to_owned());
-      digest
-    });
-    file.mut_unknown_fields().add_fixed32(42, 42);
-    file
-  });
-
-  let error = verify_directory_canonical(EMPTY_DIGEST, &directory).expect_err("Want error");
-  assert!(
-    error.contains("unknown"),
-    format!("Bad error message: {}", error)
-  );
-}
+// TODO(tonic/prost): Does Prost allow access to unknown fields?
```

Comment:
Seems not: https://github.com/danburkert/prost/issues/2

Reply:
Okay then I'm going to remove those tests.